doc-content/enterprise-only/processes/asynchronous-signal-events-con.adoc

```diff
@@ -0,0 +1,10 @@
+[id='asynchronous-signal-events-con']
+
+= Asynchronous signal events
+
+In some cases where several process instances from different process definitions are waiting for the same signal. They are generally executed sequentially in the same single thread. But, if one of those process instances throws a runtime exception, all the other process instances are affected and usually result in a rolled back transaction. To avoid this situation, {PRODUCT} supports using asynchronous signals events for:
```

Comment:
Suggested rewrite:
When several process instances from different process definitions are waiting for the same signal, they are executed sequentially in the same thread.

Reply:
Done and thank you.